src/BlazorWebView/tests/MauiDeviceTests/MauiBlazorWebView.DeviceTests.csproj

```diff
@@ -9,6 +9,7 @@
     <NoWarn>$(NoWarn),CA1416</NoWarn>
     <!-- Disable multi-RID builds to workaround a parallel build issue -->
     <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst')) and '$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)' == 'arm64'">maccatalyst-arm64</RuntimeIdentifier>
```

Comment:
Does it matter that this is checking whether the _current machine_ is ARM64? Why would that matter for what RID to target? I get that it matters if you're going to _run_ it, but it shouldn't affect compilation, no?

Reply:
Yeah, we typically build and run on the same machine on CI. The real issue is that Blazor and/or maccatalyst can do multiple rids. So without this people on arm64 can or deploy as it always builds x64.